test/blackbox-tests/test-cases/melange/javascript-extension.t

```diff
@@ -16,8 +16,7 @@ Can use extension with dots
   > (melange.emit
   >  (target output)
   >  (alias melange)
-  >  (module_system commonjs)
-  >  (javascript_extension bs.js))
+  >  (module_systems (commonjs bs.js)))
```

Comment:
I can't comment on the actual line 1, but it includes a description that is now obsolete:

https://github.com/ocaml/dune/blob/955fab5f9bced5987da8265b16f4ca2bcafcfd03/test/blackbox-tests/test-cases/melange/javascript-extension.t#L1

I also wonder if this whole test file should be removed and integrated into the new `modules-system.t`